src/tests/myFinalTests.spec.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { currentOrderOnPage, sortArray, compareTwoArrays } = require('./utils');
+
+['az', 'za', 'lohi', 'hilo'].forEach((sortingType) => {
+    test.describe('Basic tests', () => {
+        test.beforeEach(async ({ loginPage, inventoryPage }) => {
+            await loginPage.navigate();
+            await loginPage.performLogin('standard_user', 'secret_sauce');
```

Comment:
it would be better to move user credentials to the separate module

Reply:
Updated